docs/content/mongocxx-v3/installation/testing.md

```diff
@@ -0,0 +1,140 @@
++++
+date = "2020-09-26T13:47:02-04:00"
+title = "Testing"
+[menu.main]
+  identifier = "mongocxx3-installation-testing"
+  parent = "mongocxx3-installation"
+  weight = 34
++++
+
+
+### Step 6: Test your installation
```

Comment:
I'd argue for removing the "Step 6" here. This is not a required step of installation.

Reply:
Agreed.